### DISCUSSION TITLE
How to get my initial user name back?

### DISCUSSION THREAD

**Discussion by buhtz** [community/community#36958] [Other Feature Feedback, Questions, & Ideas]
My initial user name on GitHub was `buhtz` and I used it for years.
Then I played a bit around with other usernames. Now I want to get my original name back. But it is marked as "unavailable".

Can you do something about it?

Or can you tell me how long it will be blocked?

**Comment by zaahiral** [comment]
looks like it's either taken or locked, maybe contact github support

**Reply by buhtz** [reply]
> contact github support

How if I am not an enterprise user?

**Reply by airtower-luna** [reply]
The form is here: https://support.github.com/request

**Comment by AbdulAhad365** [comment]
Ask in the supprt of the Github they may help you.

**Comment by azat-io** [comment]
You can change your username in account settings:
https://github.com/settings/admin

Just click on "Change username" button.

**Reply by buhtz** [reply]
read carefully honey

**Comment by Khushboqof** [comment]
I don't think the original username was used by someone else when you updated it, or the username you used once can't be used again.

**Comment by Khushboqof** [comment]
Ask for help from github if the username is taken by someone else or blocked
